tasks/test_e2e.yml

```diff
@@ -0,0 +1,61 @@
+- name: copy shell script that executes prometheus query for collectd metrics
+  copy:
+    src: "{{ item }}"
+    dest: "/tmp"
+    owner: root
+    group: root
+    mode: 0755
+  with_items:
+    ['collectd_prometheus_query.sh']
+
+- name: execute promethues collectd query 
+  shell: "/bin/bash /tmp/collectd_prometheus_query.sh &"
```

Comment:
Does this have to be run in this manner? Or can the process be run in the foreground so that the STDOUT can be captured and used for the tests? It might be useful to know that this task ran successfully, for example, in case a failure in this would result in failures in subsequent tasks.

To make sure the command is run successfully (i.e. without errors), you can use the failed_when option for the task 
```
- name: execute promethues collectd query 
  shell: "/bin/bash /tmp/collectd_prometheus_query.sh &"
    register: checkmyconf
    failed_when:
      - checkmyconf.rc != 0
```